### ISSUE TITLE
[1.19.2 - Unique Enchantments - Utils] Bug with Climber and ladders [Fixed in development]

### ISSUE THREAD

**Issue by ShinryuAspect** [TinyModularThings/UniqueEnchantments#89]
Apparently, if I'm wearing unenchanted leggins (haven't tried with enchanted ones and it's a pair of iron ones), it seems that, while climbing a ladder, if I look upwards, I get upwards speed for no reason and auto climb it. I suppose it's an issue with the Climber enchanment affecting items, even tho they're unenchanted, for some weird reason.

**Comment by Geekerandy** [comment]
Just to be sure, you need to be wearing leggings? And it doesn’t work if you aren’t wearing leggings?

**Comment by Speiger** [comment]
Pretty sure this is addressed in the next patch.
the issue is that this patch is basically a HUGE one and we are so burned out of it that it is 8 months late at this point.
And it would be more work to just cut out the fixes and just releasing those then finishing the patch.
I hope we can finish it soon...

**Comment by ShinryuAspect** [comment]
Yep, it only appears to missbehave while wearing leggins. As soon as I unequip them, I start to fall if I don't input any movement.
And no rush with the patch. As a game developer, I know that it's inconvenient to release tiny patches literally every day and it's best partice to just merge them all while reformatting the logic if needed.

**Comment by Xaikii** [comment]
Just confirming, this is known and fixed in the next Update. 
Cause for the interaction is us leaving in the testing changes and it was active all the time as long as Leggings were worn

**Comment by ShinryuAspect** [comment]
Welp, then I'll go on and mark this as closed as it's already fixed. Thank you for taking your time and effort.

**Comment by Xaikii** [comment]
Please keep this open to avoid duplicate issue being opened :)

**Comment by ShinryuAspect** [comment]
Oh, okay, sorry XD
